src/main/java/com/ft/methodeimagemodelmapper/service/GraphicResolver.java

```diff
@@ -0,0 +1,50 @@
+package com.ft.methodeimagemodelmapper.service;
+
+import com.ft.methodeimagemodelmapper.model.EomFile;
+import com.googlecode.pngtastic.core.PngException;
+import com.googlecode.pngtastic.core.PngImage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedInputStream;
+import java.io.ByteArrayInputStream;
+import java.nio.charset.Charset;
+import java.util.Optional;
+
+import static com.ft.methodeimagemodelmapper.service.MethodeImageModelMapper.IMAGE_TYPE;
```

Comment:
using this constant creates a dependency between the `GraphicsResolver` and the `MethodeImageModelMapper`. However the `MethodeImageModelMapper` uses the `GraphicsResolver`. So, simply in order to re-use a part of a constant we have a cyclic dependency between these two classes. I'd rather just dupe the constant or extract the reusable constants to a common class / interface